subsys/bluetooth/mesh/rpl_strg_settings.c

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
From my personal experience, when names of files and API functions are aligned, it is much easier to navigate in the code:
- `rpl_storage.h`, `rpl_storage_settings.c`, `rpl_storage_emds.c`, `bt_mesh_rpl_storage_...`, or
- `rpl_strg.h`, `rpl_strg_settings.c`, `rpl_strg_emds.c`, `bt_mesh_rpl_strg_...`;